ports/stm/boards/stm32f411ce_blackpill_haxpress/mpconfigboard.mk

```
@@ -0,0 +1,18 @@
+USB_VID = 0x239A
+USB_PID = 0x806A
```

Here's a new PID we assigned for this:
```suggestion
USB_PID = 0x006A
```

---

Not sure how to mark this resolved.